src/Bookings.js

```diff
@@ -12,7 +12,7 @@ const Bookings = () => {
     <div className="App-content">
       <div className="container">
         <Search search={search} />
-        {/* <SearchResults results={FakeBookings} /> */}
```

Comment:
You may remove this line since you don't need it!

Reply:
Right, thank you